Lib/test/test_types.py

```diff
@@ -711,6 +712,11 @@ def call(part):
         """
         assert_python_ok("-c", code)
 
+    def test_frame_locals_proxy(self):
+        frame = inspect.currentframe()
+        self.assertIsNotNone(frame)
+        self.assertIsInstance(frame.f_locals, types.FrameLocalsProxyType)
```

Comment:
Test properties of the type itself?

I would also move this to below `test_capsule_type`, so that all the type tests are in the same place.

```suggestion
    def test_frame_locals_proxy_type(self):
        self.assertIsInstance(types.FrameLocalsProxyType, type)
        self.assertIsInstance(types.FrameLocalsProxyType.__doc__, str)
        self.assertEqual(types.FrameLocalsProxyType.__module__, 'types')

        frame = inspect.currentframe()
        self.assertIsNotNone(frame)
        self.assertIsInstance(frame.f_locals, types.FrameLocalsProxyType)
```